optimize_sidebars.js

```diff
@@ -133,19 +167,13 @@ module.exports = {
               "Common pitfalls",
               "components/console/console-troubleshooting/common-pitfalls/"
             ),
-            docsLink(
-              "Feedback and support",
-              "components/console/console-troubleshooting/feedback-and-support/"
-            ),
           ],
         },
       ],
     },
 
     {
       Modeler: [
-        docsLink("About Modeler", "components/modeler/about-modeler/"),
```

Comment:
This is an unfortunate change, but not something we can resolve before the next release. 

We have several pages like this, which we converted to "category" pages. They include a bit of content, and also list the pages inside them. Within the left nav, they appear as an expandable category, but when you click on them it takes you to the category page instead of only expanding.

Docusaurus does not support "category" types that don't point at a document within the current instance. In this example, this is the sidebars file for the Optimize instance....and this category page lives within the main docs instance. We can't put them in the sidebars as a simple link, or they'll make the sidebars look out of sync. 

I think the long-term answer is to work with Docusaurus to support category pages that point at URLs instead of document IDs. But that's not happening as part of this release.

Reply:
I'm not understanding how this will function. Should we avoid these category pages?

Reply:
I don't think we need to avoid them. Engineers seem to be happy about them. There is a little difference in how each of the instances (main vs optimize) handle them, but that's no different than the large amounts of sidebar drift we get in between releases. Clicking across instances in the left nav is something that I do pretty regularly to test the cross-instance functionality, but I don't think any actual users do it very often.

How this functions right now: 

In the original instance, the category page shows up as a navigable category. It not only expands, but you also go to a page, which usually enumerates the pages within that category. Example: https://docs.camunda.io/docs/components/modeler/about-modeler/ -- if you view another page next to it, then click back on Modeler, you'll see that it takes you to the category page.

In the opposite instance, the category page will show up as a category, but it will **not** be navigable. Meaning you can expand it in the tree, but you can't actually visit that page. Example: https://docs.camunda.io/optimize/components/what-is-optimize/ -- expand the Modeler option in the left nav, and you'll see that it functions differently than it did when you were in the Modeler section.

Long-term, if we work with Docusaurus to see if they're interested in supporting non-instance URLs in category sidebar items, we could make the latter work just like the former.